clientconn_state_transition_test.go

```diff
@@ -41,78 +43,136 @@ func init() {
 	balancer.Register(testBalancer)
 }
 
+// These tests use a PipeListener. This listener is similar to net.Listener except that it is unbuffered, so each read
+// and write will wait for the other side's corresponding write or read.
 func TestStateTransitions_SingleAddress(t *testing.T) {
+	defer leakcheck.Check(t)
+
+	mctBkp := getMinConnectTimeout()
+	// Call this only after transportMonitor goroutine has ended.
```

Comment:
"transportMonitor goroutine"?  What's that?

Reply:
Whoops, done.